src/Workspaces/Core/Portable/FindSymbols/SymbolFinder_Hierarchy.cs

```diff
@@ -19,31 +19,39 @@ namespace Microsoft.CodeAnalysis.FindSymbols
 {
     public static partial class SymbolFinder
     {
-        internal static async Task<ImmutableArray<SymbolAndProjectId>> FindOverridesAsync(
-            SymbolAndProjectId symbolAndProjectId, Solution solution, IImmutableSet<Project> projects = null, CancellationToken cancellationToken = default)
+        /// <summary>
+        /// Find symbols for members that override the specified member symbol.
+        /// </summary>
+        public static async Task<IEnumerable<ISymbol>> FindOverridesAsync(
+            ISymbol symbol, Solution solution, IImmutableSet<Project> projects = null, CancellationToken cancellationToken = default)
+        {
+            return await FindOverridesArrayAsync(symbol, solution, projects, cancellationToken).ConfigureAwait(false);
+        }
+
+        internal static async Task<ImmutableArray<ISymbol>> FindOverridesArrayAsync(
+            ISymbol symbol, Solution solution, IImmutableSet<Project> projects = null, CancellationToken cancellationToken = default)
```

Comment:
note: we still have an internal/public split here with the public guy boxing the ImmutableArray into an IEnumerable.  Not sure what we'd like to do moving forward.